src/pattern-library/examples/popover-basic.tsx

```diff
@@ -0,0 +1,27 @@
+import { useRef, useState } from 'preact/hooks';
+
+import { Popover } from '../../components/feedback';
+import { Button } from '../../components/input';
+import { useClickAway } from '../../hooks/use-click-away';
+
+export default function App() {
+  const [open, setOpen] = useState(false);
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
+
+  useClickAway(buttonRef, () => setOpen(false));
```

Comment:
I added this because popovers handled via the popover API are automatically hidden when clicking away, which makes the `open` state no longer reflect reality.

I didn't notice this first because in the case of `Select` we have `useClickAway`, `useFocusAway`, etc.

I will address this separately, but we'll probably need to add an `onClose` callback to the `Popover` and handle a few native events, like `toggle` one.

Reply:
> I added this because popovers handled via the popover API are automatically hidden when clicking away, which makes the open state no longer reflect reality.

This seems like an obvious hazard that we should at least document.

> I will address this separately, but we'll probably need to add an onClose callback to the Popover and handle a few native events, like toggle one.

Yes, definitely. Is there an issue tracking this?

Reply:
> Is there an issue tracking this?

Done https://github.com/hypothesis/frontend-shared/issues/1799